automated-comments/go/two-fer/missing_share_with_function.md

```diff
@@ -0,0 +1 @@
+The `ShareWith` function is missing or couldn't be found.
```

Comment:
Maybe just "is missing"?

Reply:
Actually, per the guidelines, I think this needs an action:

```suggestion
 Add the function `ShareWith`, as it is required and currently missing.
```

If you want action - cause;

```suggestion
The `ShareWith` function is required and missing. Add the function `ShareWith`.
```

If you want the reverse (ok in this instance, IMO).

Or something like that.